src/services/auth.service.js

```diff
@@ -79,3 +84,34 @@ function register(name, email, phone, username, password1, password2) {
             }
         });
 }
+function chechUser(userId) {
+    console.log(userId);
+     return axios.get(
+        stringFormat(`${config.commonConfig.$apiUrl}/${config.userConfig.api.checkUser.endpoint}`, userId)
+    )
+
+        .then((response) =>{
+            console.log('response =', response.data);
+        })
+        .catch((error) => {
+            console.log('error =', error.data)
+        }) 
+}
+
+function sendOtp() {
+    return axios.post(
+        stringFormat(`${config.commonConfig.$apiUrl}/${config.userConfig.api.sendOtp.endpoint}`, username)
+   )  
+}
+
+function verifyOtp(otp){
+    return axios.put(
+        stringFormat(`${config.commonConfig.$apiUrl}/${config.userConfig.api.verifyOtp.endpoint}`, username, otp)
+    )
+}
+
+function resetPassword(otp){
+    return axios.put(
+        `${config.commonConfig.$apiUrl}/${config.userConfig.api.resetPassword.endpoint}`, otp
```

Comment:
- `password` (to be accepted in function argument)
- `password` will be sent in JSON data body as done in login or registration.
```
{"password": password}
```